src/meta/src/hummock/manager/mod.rs

```diff
@@ -1294,24 +1306,22 @@ impl HummockManager {
         false
     }
 
-    #[named]
     pub async fn report_compact_task(
         &self,
         task_id: u64,
         task_status: TaskStatus,
         sorted_output_ssts: Vec<SstableInfo>,
         table_stats_change: Option<PbTableStatsMap>,
     ) -> Result<bool> {
-        let mut guard = write_lock!(self, compaction).await;
-        self.report_compact_task_impl(
-            task_id,
-            None,
-            task_status,
-            sorted_output_ssts,
-            &mut guard,
-            table_stats_change,
-        )
-        .await
+        let rets = self
+            .report_compact_tasks(vec![ReportTask {
+                task_id,
+                task_status: task_status as i32,
+                sorted_output_ssts,
+                table_stats_change: table_stats_change.unwrap_or_default(),
+            }])
+            .await?;
+        Ok(rets[0])
```

Comment:
Should we return Vec\<bool\> instead of rets[0]? Assuming that we ensure that all report tasks are successful, then this return value is meaningless